store.go

```diff
@@ -2920,6 +2979,24 @@ func (s *store) Layers() ([]Layer, error) {
 		return nil, err
 	}
 
+	return s.appendROLayers(layers)
+
+}
+
+func (s *store) Layers() ([]Layer, error) {
+	lstore, err := s.ROLayerStore()
+	if err != nil {
+		return nil, err
+	}
+	layers, err := lstore.Layers()
```

Comment:
How does `store.Layers()` notice changes in the primary layers list?

Reply:
I think they're independent right?